content/class/Media.md

```diff
@@ -0,0 +1,213 @@
+**Project NewCaw** \
+Author: Frederick Schenk
+
+# Content/Media
+
+![API: Twitter 1](https://img.shields.io/badge/API-Twitter%201-lightgrey?style=flat-square) ![API: Twitter 2](https://img.shields.io/badge/API-Twitter%202-blue?style=flat-square) ![API: Mastodon](https://img.shields.io/badge/API-Mastodon-purple?style=flat-square)
+
+```c#
+public interface Media
+```
+
+`Media` represents a generic interface for media contained in a [`Post`](Post.md). It is subclassed by classes which implements actual media types such as images or video.
+
+| API    | Twitter 1 | Twitter 2 | Mastodon |
+| ------ | --------- | --------- | -------- |
+| Object | [Media](https://developer.twitter.com/en/docs/twitter-api/v1/data-dictionary/object-model/entities#media) | [Media](https://developer.twitter.com/en/docs/twitter-api/data-dictionary/object-model/media) | [Attachment](https://docs.joinmastodon.org/entities/attachment/) |
+
+## Properties
+
+### `id` {#property_id}
+
+![API: Twitter 1](https://img.shields.io/badge/API-Twitter%201-lightgrey?style=flat-square) ![API: Twitter 2](https://img.shields.io/badge/API-Twitter%202-blue?style=flat-square) ![API: Mastodon](https://img.shields.io/badge/API-Mastodon-purple?style=flat-square)
+
+```c#
+public string id { get; }
+```
+
+The unique identifier for this media object.
+
+| API   | Twitter 1 | Twitter 2   | Mastodon |
+| ----- | --------- | ----------- | -------- |
+| Field | `id_str`  | `media_key` | `id`     |
+| Type  | string    | string      | string   |
+
+### `media_url` {#property_media_url}
+
+![API: Twitter 1](https://img.shields.io/badge/API-Twitter%201-lightgrey?style=flat-square) ![API: Twitter 2 (Not yet)](https://img.shields.io/badge/API-Twitter%202%20(Not%20yet)-darkred?style=flat-square) ![API: Mastodon](https://img.shields.io/badge/API-Mastodon-purple?style=flat-square)
```

Comment:
I'm surprised that this (and some other bits) aren't in Twitter v2 yet. But then I've not looked at the API docs in much depth.

Reply:
You can already get the media url for Pictures over an inofficial argument, but this is not yet included for videos. So, the v2 API is definitely "Work in Progress".